ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1346,6 +1346,34 @@ port_channel_interfaces:
         key: "< encrypted_password >"
 ```
 
+#### Tunnel Interfaces
+
+```yaml
+tunnel_interfaces:
+- name: < interface_name >
+  description: < description >
+  shutdown: < true | false >
+  mtu: < 68-65535 >
+  vrf: < vrf_name >
+  ip_address: < ip_address >
+  ipv6_enable: < true | false >
+  ipv6_address: < ipv6_address >
+  access_group_in: < access_list_name >
+  access_group_out: < access_list_name >
+  ipv6_access_group_in: < access_list_name >
+  ipv6_access_group_out: < access_list_name >
+  tcp_mss_ceiling:
+     ipv4: < 64-65495 >
```

Comment:
Being pedantic here, I know. But you are indenting this section 3 instead of 2.

Reply:
Addressed in 0b8ddf0